index.js

```diff
@@ -51,8 +55,17 @@ function propertyIsUnsafe(target, key) {
 			&& Object.propertyIsEnumerable.call(target, key)) // and also unsafe if they're nonenumerable.
 }
 
+// Retrieves either a new object or the appropriate target object to mutate.
+function getDestinationObject(target, options) {
+	if (options && !!options.mergeWithTarget) {
```

Comment:
The `!!` isn't needed.
`if` statements don't care if you pass them a boolean or truthy/falsy value.

Reply:
Good call. Fixed.